### PR TITLE
fix(docs): update circleci remote cache docs

### PR DESCRIPTION
**Pull request by tknickman**
Updates docs to reflect findings from https://github.com/vercel/turborepo/issues/1343

> This appears to be an issue with how CircleCI environment variables work. 
Because we're setting the environment variables [on the project](https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-project), you actually don't need to specify them in the workflow. Adding these in the workflow is overriding the variables set on the project [because they have a higher precedence](https://circleci.com/docs/2.0/env-vars/#order-of-precedence), so the environment variables literally end up being the strings $TURBO_TEAM and $TURBO_TOKEN, and your remote cache requests are failing to authenticate.

### CODE REVIEW

**Comment by tknickman** [comment]
@mkotsollaris agreed, this has been a pain point with the remote cache config - I do have a few ideas about how to make this easier / automated.

**Comment by jaredpalmer** [comment]
This is a good suggestion. @tknickman can you update to use Vercel URL instead of slug?

**Comment by tknickman** [comment]
Yes we could either make this just the full URL and we can parse out the slug ourselves, or provide an automatic method of retrieving the slug.

I do like just using the full url.

**Comment by tknickman** [comment]
Updated the docs to be more explicit about where to retrieve the value for `TURBO_TEAM`

File: docs/pages/docs/ci/circleci.mdx [hunk 1]
```diff
@@ -151,21 +139,10 @@ Copy the value to a safe place. You'll need it in a moment.
 ![CircleCI Environment Variables](/images/docs/circleci-environment-variables.png)
 ![CircleCI Create Environment Variables](/images/docs/circleci-create-environment-variables.png)
 
-3. Make a second secret called `TURBO_TEAM` and enter the value of your team's Vercel URL (or if you're on Hobby, your personal URL works as well). Do not include the `https://vercel.com/` part, only the slug.
+3. Make a second secret called `TURBO_TEAM` and enter the value of your team's Vercel URL _without_ the `vercel.com/`. Your Team URL can be found inside your team's general project settings from the dashboard.
+
+   If you're on a Hobby, you can use your username. Your username can be found in your [Vercel Personal Account Settings](https://vercel.com/account)
```
Comment (nathanhammond) [inline docs/pages/docs/ci/circleci.mdx:144]:
```suggestion
   If you're using a Hobby Plan, you can use your username. Your username can be found in your [Vercel Personal Account Settings](https://vercel.com/account)
```

Or `in a Hobby Project`?

File: docs/pages/docs/ci/github-actions.mdx [hunk 2]
```diff
@@ -213,7 +213,9 @@ Copy the value to a safe place. You'll need it in a moment.
 ![GitHub Secrets](/images/docs/github-actions-secrets.png)
 ![GitHub Secrets Create](/images/docs/github-actions-create-secret.png)
 
-3. Make a second secret called `TURBO_TEAM` and enter the value of your team's Vercel URL (or if you're on Hobby, your personal URL works as well). Do not include the `https://vercel.com/` part, only the slug.
+3. Make a second secret called `TURBO_TEAM` and enter the value of your team's Vercel URL _without_ the `vercel.com/`. Your Team URL can be found inside your team's general project settings from the dashboard.
+
+   If you're on a Hobby, you can use your username. Your username can be found in your [Vercel Personal Account Settings](https://vercel.com/account)
```
Comment (nathanhammond) [inline docs/pages/docs/ci/github-actions.mdx:218]:
```suggestion
   If you're using a Hobby Plan, you can use your username. Your username can be found in your [Vercel Personal Account Settings](https://vercel.com/account)
```

Or `in a Hobby Project`?

File: docs/pages/docs/ci/gitlabci.mdx [hunk 3]
```diff
@@ -132,6 +132,8 @@ Copy the value to a safe place. You'll need it in a moment.
 ![GitLab CI Variables](/images/docs/gitlab-ci-variables.png)
 ![GitLab CI Create Variable](/images/docs/gitlab-ci-create-variable.png)
 
-3. Make a second secret called `TURBO_TEAM` and enter the value of your team's Vercel URL (or if you're on Hobby, your personal URL works as well). Do not include the `https://vercel.com/` part, only the slug.
+3. Make a second secret called `TURBO_TEAM` and enter the value of your team's Vercel URL _without_ the `vercel.com/`. Your Team URL can be found inside your team's general project settings from the dashboard.
+
+   If you're on a Hobby, you can use your username. Your username can be found in your [Vercel Personal Account Settings](https://vercel.com/account)
```
Comment (nathanhammond) [inline docs/pages/docs/ci/gitlabci.mdx:137]:
```suggestion
   If you're using a Hobby Plan, you can use your username. Your username can be found in your [Vercel Personal Account Settings](https://vercel.com/account)
```

Or `in a Hobby Project`?

File: docs/pages/docs/ci/travisci.mdx [hunk 4]
```diff
@@ -116,7 +116,9 @@ Copy the value to a safe place. You'll need it in a moment.
 
 ![Travis CI Variables](/images/docs/travis-ci-environment-variables.png)
 
-3. Make a second secret called `TURBO_TEAM` and enter the value of your team's Vercel URL (or if you're on Hobby, your personal URL works as well). Do not include the `https://vercel.com/` part, only the slug.
+3. Make a second secret called `TURBO_TEAM` and enter the value of your team's Vercel URL _without_ the `vercel.com/`. Your Team URL can be found inside your team's general project settings from the dashboard.
+
+   If you're on a Hobby, you can use your username. Your username can be found in your [Vercel Personal Account Settings](https://vercel.com/account)
```
Comment (nathanhammond) [inline docs/pages/docs/ci/travisci.mdx:121]:
```suggestion
   If you're using a Hobby Plan, you can use your username. Your username can be found in your [Vercel Personal Account Settings](https://vercel.com/account)
```

Or `in a Hobby Project`?

**Comment by nathanhammond** [comment]
[> This is a good suggestion. @tknickman can you update to use Vercel URL instead of slug?](https://github.com/vercel/turborepo/pull/1345#issuecomment-1148832180) — @jaredpalmer

The control on the Vercel settings page is segmented so there is no easy way to copy/paste the full URL. Supporting the full URL is likely only going to introduce more edge cases:
- Do we require the `https`?
- What if they add a `www`?
- Ignore a trailing slash?

Screenshot 2022-06-08 at 09-11-45 General – Account – Dashboard – Vercel

If we're trying to remove the papercuts I'd rather set up integration to [handle doing this for the user](https://circleci.com/docs/api/v2/#operation/addEnvironmentVariableToContext) instead.

**Review by nathanhammond — APPROVED** [review]
🛠🚢: just one language issue, made it so you could accept suggestions if you're happy with my proposal.

**Pull request merged**